### PR TITLE
Mat 6665 sde highlighing sub tab

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
## MADiE PR

Jira Ticket: [MAT-6665](https://jira.cms.gov/browse/MAT-6665)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included
* [x] No extemporaneous files are included (i.e Complied files or testing results)
* [x] This PR is into the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [x] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: src/components/editTestCase/groupCoverage/QdmGroupCoverage.tsx [hunk 1]
```diff
@@ -55,11 +56,20 @@ const definitionFilterCondition = (statementResult, definitionCategory) => {
   );
 };
 
+const sdeFilter = (statementResult, definitionCategory) => {
```
Comment (RohitKandimalla) [inline src/components/editTestCase/groupCoverage/QdmGroupCoverage.tsx]: Looks like definitionCategory is never used

File: src/components/editTestCase/groupCoverage/QdmGroupCoverage.tsx [hunk 2]
```diff
@@ -25,6 +25,7 @@ interface Props {
   measureGroups: Group[];
   groupCoverageResult: GroupCoverageResult;
   cqlDefinitionCallstack;
+  includeSDE;
```
Comment (RohitKandimalla) [inline src/components/editTestCase/groupCoverage/QdmGroupCoverage.tsx]: Can we type the properties, especially when declaring in an interface ?

Comment (sb-cecilialiu) [reply src/components/editTestCase/groupCoverage/QdmGroupCoverage.tsx]: @RohitKandimalla added boolean type.

File: src/components/editTestCase/groupCoverage/QdmGroupCoverage.tsx (hunk 1 above)
Comment (sb-cecilialiu) [reply src/components/editTestCase/groupCoverage/QdmGroupCoverage.tsx]: @RohitKandimalla I used it before but forgot to remove it, good catch, thanks just removed it.

**Review by RohitKandimalla — DISMISSED** [review]

**Review by RohitKandimalla — APPROVED** [review]

**Pull request merged**